tests/minimal/Taskfile.yml

```diff
@@ -0,0 +1,34 @@
+# https://taskfile.dev/#6/installation
+version: 3
+
+includes:
+  windows: ./Taskfile_windows.yml
+
+tasks:
+  build:
+    - cmake . -B ./build -DCMAKE_BUILD_TYPE:STRING=Debug -G '{{.CMAKE_GENERATOR | default "Ninja Multi-Config"}}' {{.CMAKE_ARGS}}
+    - cmake --build ./build --config Debug
+
+  build:mingw:
+    cmds:
+      - task: build
+        vars:
+          CMAKE_ARGS: -DENABLE_CROSS_COMPILING:BOOL=ON -DCMAKE_C_COMPILER={{.CROSS_CC | default "x86_64-w64-mingw32-gcc"}} -DCMAKE_CXX_COMPILER={{.CROSS_CXX | default "x86_64-w64-mingw32-g++"}}
+
+  build:mingw:from-env:
+    env: 
+      CC: x86_64-w64-mingw32-gcc
+      CXX: x86_64-w64-mingw32-g++
+    cmds:
+      - task: build
+        vars:
+          CMAKE_ARGS: -DENABLE_CROSS_COMPILING:BOOL=ON
+
+  build:mingw:from-triplet:
```

Comment:
I prefer `.` as a name separator and not `:`

Reply:
Any other name conventions or style changes ?

_Would be nice to write down some Guidelines into CONTRIBUTING.md, some other differences from [Taskfile Styleguide](https://taskfile.dev/styleguide/)._  
_I think everything else can be linted._

**Note:**
> Use colon for task namespacing
>
> This is also done automatically when using included Taskfiles.

https://taskfile.dev/styleguide/#use-colon-for-task-namespacing

_Not sure if this breaks the style a bit when using includes: `minimal:build.mingw.from-triplet` vs `minimal:build:mingw:from-triplet`_

Reply:
Yeah, it breaks the specified style, but I prefer using `.`. To me using `:` is kind of strange. I haven't seen any programming language using that for accessing children.

Reply:
It seems that `:` here means the sub-tasks of a Taskfile. I wish they had used `.` for that too.